src/main/java/de/neuefische/ProductRepo/ProductDB.java

```diff
@@ -0,0 +1,33 @@
+package de.neuefische.ProductRepo;
```

Comment:
Ich glaube hier gab es ein Versehen - 

`ProductRepo` ist die gewünschte Klasse die wir haben wollen.
Das Package in dem wir es erzeugen würden, hieße `product`.

In `product` würden wir alles zusammenbringen das mit Produkten zutun hat.
Also `Product` und `ProductRepo`. Das Gleiche dann für `Order`.
Insgesamt könnte es dann so aussehen

![image](https://user-images.githubusercontent.com/23424538/190156120-ad25ca56-dae7-48f5-8f16-92f80cbc2fd8.png)

Reply:
Ja, das hat sich mir erst jetzt so vollkommen erschlossen, da wir jetzt in dieser Woche in die Strukturierung Controller/Service/Repo eingetaucht sind... Jetzt ist die gesamte Strukturierung eher verständlich geworden...

Reply:
Prima, dann wünsche ich viel Erfolg bei der nächsten Aufgabe - jetzt kommt der richtig spannende Teil :smirk: